package.json

```diff
@@ -51,6 +52,17 @@
     "@commitlint/cli": "16.0.2",
     "@commitlint/config-conventional": "16.0.0",
     "@craco/craco": "6.4.3",
+    "@storybook/addon-actions": "^6.4.22",
```

Comment:
Is it unnecessary to freeze the versions here and below?

Reply:
fixed